project/Doc.scala

```diff
@@ -166,7 +166,7 @@ object BootstrapGenjavadoc extends AutoPlugin {
   override def requires =
     UnidocRoot.CliOptions.genjavadocEnabled
       .ifTrue {
-        val onJdk8 = System.getProperty("java.version").startsWith("1.")
+        val onJdk8 = JavaVersion.is1x
```

Comment:
This now implies that `is1x` means Java 8, so better call it `is8` then?

Reply:
@johanandren @raboof Shouldn't it actually be more like: if java.specification.version < 11 based on the requirement message or is it actually just requiring 8 and the message needs to be updated?